README.md

```diff
@@ -101,6 +101,16 @@ Compliance levels can also be updated through the `testing.accessibility.complia
 
 The test file lives in `/tests/accessibility/compliance/pa11y.js` if any edits are needed (such as staging credentials, if you're running tests in an environment that requires authentication).
 
+### Visual Regression Testing
```

Comment:
Can you add something about using the custom scenarios and what the config files are for? Mention the breakpoints as well, that they should align with the breakpoints for your project. Anything someone might be updating for their project should be documented in the readme so they don't have to guess and risk burning time breaking/fixing something.

Reply:
Also something about how to add new URLs for testing the package file.